lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -101,6 +104,8 @@ def trigger
       puppet_trigger['start_date'] = self.class.normalized_date("#{trigger['start_year']}-#{trigger['start_month']}-#{trigger['start_day']}")
       puppet_trigger['start_time'] = self.class.normalized_time("#{trigger['start_hour']}:#{trigger['start_minute']}")
       puppet_trigger['enabled']    = trigger['flags'] & Win32::TaskScheduler::TASK_TRIGGER_FLAG_DISABLED == 0
+      puppet_trigger['minutes_interval'] = trigger['minutes_interval'] ||= 0
+      puppet_trigger['minutes_duration'] = trigger['minutes_duration'] ||= 0
```

Comment:
IIUC, `minutes_interval` and `minutes_duration` are only meaningful for `daily` triggers? In which case, I'd prefer to move this logic up to the `when Win32::TaskScheduler::TASK_TIME_TRIGGER_DAILY` block